test/aircraft/Pilot/Pilot.spec.js

```diff
@@ -409,6 +401,29 @@ ava('.climbViaSID() correctly configures MCP and returns correct response when a
     t.true(pilot._mcp.altitude === 11000);
 });
 
+ava('.cross() correctly configures MCP and returns correct response ', (t) => {
+    const aircraftModel = new AircraftModel(ARRIVAL_AIRCRAFT_INIT_PROPS_MOCK);
+    const pilot = createPilotFixture();
```

Comment:
I'm going to nit-pick on naming a little bit here:

* things that are fixtures should be suffixed with the word `Fixture`.  so `pilot` -> `pilotFixture`
* things that are mocks, should similarly be suffixed with the word `Mock`.  so `fixName` -> `fixNameMock` and `altitude` -> `altitudeMock`

My guess is that this convention isn't followed everywhere, which is ok for now.  Going forward I want to try and be consistent.  As I'm sure you've discovered already, I tend to lean on the side of verbosity.

Reply:
I absolutely dislike those verbose names. IMHO they don't provide any value and make code harder to read. It's like a modern version of the Hungarian notation. 

done...